src/gsi/m_obsNodeTypeManager.F90

```diff
@@ -250,6 +259,7 @@ module m_obsNodeTypeManager
     enumerator:: iobsNode_dbz
     enumerator:: iobsNode_fed
 
+
```

Comment:
Remove extra blank line.

Reply:
Done!

Reply:
The extra blank [line](https://github.com/KariA-Spire/GSI/blob/2aba9bc75fc4ee120e122a7ac0863f4851ee26b0/src/gsi/m_obsNodeTypeManager.F90#L260) remains.

Reply:
The extra blank line remains.

Reply:
Removed!